### PR TITLE
Enhanced User API responses and fixed duplicated handlers

### PR DESCRIPTION
**Pull request by mateusz-garlacz**

### CODE REVIEW

File: cs3api4lab/handlers.py [hunk 1]
```diff
@@ -271,8 +271,7 @@ def get(self):
     (r"/api/cs3/public/shares/list", ListPublicSharesHandler),
     (r"/api/cs3/public/share", GetPublicShareByTokenHandler),
     (r"/api/cs3/ocm", OcmSharesHandler),
-    (r"/api/cs3/ocm/received", OcmReceivedSharesHandler)
-    (r"/api/cs3/public/share", GetPublicShareByTokenHandler),
```
Comment (diocas) [inline cs3api4lab/handlers.py:275]: I don't understand what you're doing here.. Why are you removing the public links endpoint?
What is this PR trying to solve?

Comment (JarCz) [reply cs3api4lab/handlers.py:275]: This commit delete duplicate line: 
`(r"/api/cs3/ocm/received", OcmReceivedSharesHandler)`
and fixes error: 
`
File "\cs3api4lab\__init__.py", line 5, in <module> from .handlers import handlers
File "\cs3api4lab\handlers.py", line 274, in <module> (r"/api/cs3/ocm/received", OcmReceivedSharesHandler)
TypeError: 'tuple' object is not callable
`

**Review by diocas — APPROVED** [review]

**Comment by diocas** [comment]
Please rebase the branch otherwise I cannot merge it. (and rebase it instead of merging on into another)

**Comment by diocas** [comment]
Guys, you ***really*** need to start doing rebases.. This way of merging master is a mess and not how you should be doing...

**Pull request merged**